src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -297,11 +302,17 @@ def as_text(self):
         Build the text body published on reporters
         '''
         message = self.message
+        reason = self.reason
+
         if len(message) > 0:
             message = message[0].capitalize() + message[1:]
-        body = '{}: {} [clang-tidy: {}]'.format(
+        if len(reason) > 0:
+            reason = reason[0].capitalize + reason[1:]
+
+        body = '{}: {} {} [clang-tidy: {}]'.format(
             self.type.capitalize(),
             message,
+            reason,
```

Comment:
Nit: I don't think you want to cram the `reason` field into the first line of the comment. Instead, please add the reason after `self.body` (see below), with a similar conditional check (i.e. `if self.reason: body += '\n{}'.format(self.reason)`).